README.md

```diff
@@ -92,15 +92,15 @@ All packages are based on the static binary bundles provided by the CRI-O CI.
 [![release-1.29](https://img.shields.io/badge/prerelease-release--1.29-blue?logo=git&logoColor=white)](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:prerelease:v1.29)
 [![release-1.28](https://img.shields.io/badge/prerelease-release--1.28-blue?logo=git&logoColor=white)](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o:prerelease:v1.28)
 
-### Distributions using `rpm` packages
-
 #### Define the Kubernetes version and used CRI-O stream
 
 ```bash
 KUBERNETES_VERSION=v1.30
-PROJECT_PATH=prerelease:/main
+CRIO_STREAM=stable:/v1.30
```

Comment:
Better to use "v1.30" here, and stable:/ below ?
Or use the full project path for both of them

Reply:
Just thinking that since they use similar OBS setups, the variables and URL templates should be similar too

Reply:
Yes, I'm changing the path and set it to `CRIO_VERSION=v1.30`.